sink-server/cors.go

```diff
@@ -1,11 +1,12 @@
 package server
 
 import (
-	"go.uber.org/zap"
 	"net/http"
 	"net/url"
 	"time"
 
+	"go.uber.org/zap"
+
 	"github.com/rs/cors"
```

Comment:
```suggestion
	"go.uber.org/zap"
    "github.com/rs/cors"
```

And a another round of `goimports`.

Reply:
Done. Thanks.